hw1/task1/vectors.py

```diff
@@ -0,0 +1,25 @@
+import math
+
+from typing import List, Union
+
+Vector = List[Union[int, float]]
```

Comment:
Actually, you do not need to write `Union[int, float]`.

https://www.python.org/dev/peps/pep-0484/#the-numeric-tower

`float`  already "implies" that `int` can be given, and raises no errors. So you can just keep `float`.

Reply:
(type aliases are cool, though)